JSONRenderer.php

```diff
@@ -44,6 +47,14 @@ public function renderReport(Report $report)
                 );
             }
 
+            $name = $violation->getMethodName() ?: $violation->getFunctionName() ?: $violation->getClassName();
```

Comment:
Looks like there's a dedicated "Null Coalescing" operator, which is what I really want here. Will update